addons/l10n_fi/__manifest__.py

```diff
@@ -22,7 +22,7 @@
 Set the payment reference type from the Sales Journal.
     """,
     "depends": [
-        'account',
+        'l10n_multilang',
```

Comment:
Can't change the dependency in stable

Reply:
I will retarget the PR in 16.2 since the translation was just a backport and i don't really want to retranslate the new account in finnish in 16.0, thanks i will remember this for next time ahah